monix-eval/shared/src/main/scala/monix/eval/Coeval.scala

```diff
@@ -74,35 +74,35 @@ import scala.util.{Failure, Success, Try}
   * instances out of strict values:
   *
   * {{{
-  *   val fa = Coeval.now(1)
-  *   fa.value //=> 1
+  *   val faNow = Coeval.now(1)
+  *   faNow.value //=> 1
```

Comment:
Nitpick 1: I'd prefer `// =>` style, if that's possible
Nitpick 2: `value` now uses parens, so it should be `value()`.

Reply:
that should be easy